source/grid/grid_tools.cc

```diff
@@ -927,7 +939,6 @@ namespace GridTools
   }
 
 
-
```

Comment:
undo this line deletion

Reply:
No, please make sure there are 3 empty lines (so add 2 lines you removed earlier).